en/examples/auth-external-jwt.md

```diff
@@ -1,33 +1,36 @@
 ---
-title: Auth External API (JWT)
-description: Authentication with external API service (jsonwebtoken) example with Nuxt.js
+title: API d'authentification externe (JWT)
+description: Exemple d'authentification avec le service d'API externe (jsonwebtoken) avec Nuxt.js
 github: auth-jwt
 code: https://github.com/ahadyekta/nuxt-auth-external-jwt
 ---
 
 # Documentation
 
-<p style="width: 294px;position: fixed; top : 64px; right: 4px;" class="Alert Alert--orange"><strong>⚠Cette page est actuellement en cours de traduction française. Vous pouvez repasser plus tard ou <a href="https://github.com/vuejs-fr/nuxt" target="_blank">participer à la traduction</a> de celle-ci dès maintenant !</strong></p><p>In auth-routes example both api and nuxt start together and use one node server. However, sometimes we should work with external api with jsonWebToken. In this example it will be explained in a simple way.</p>
+Dans l'exemple auth-routes l'API et le site Nuxt se lance ensemble et utilisent la même instance serveur Node.js. Cependant, il est parfois mieux de travailler avec une API externe avec jsonWebToken. Dans cet exemple il sera expliqué comment faire cela simplement.
 
-## Structure:
+## Structure
 
-Since the nuxt.js provides both server and client rendering and the cookie of browser is different from cookie of the node server, we should push token data to somestorage that can be accessible in both sides.
+Puisque Nuxt.js fournit à la fois le rendu client et serveur ainsi qu'un cookie différent entre le navigateur et le serveur Node.js, nous devons fournir un jeton de donnée qui puisse être accessible par les deux parties.
 
-### For server rendering:
-We should save the token in browser cookie after login, then it can be accessed through `req.headers.cookie` in middleware files, nuxtServerInit function or  wherever you can access the `req`.
-### For client rendering:
-We directly commit token in the store, as long as the page is not closed or reloaded, we have the token.
+### Pour le rendu serveur
 
-First, we install the dependencies:
+Nous devons sauvegarder le jeton dans un cookie de session après connexion qui puisse être récupéré via `req.headers.cookie` par les fichiers middlewares, la fonction `nuxtServerInit` ou tout ce qui a accès à `req`.
+
+### Pour le rendu client
+
+Nous actons directement le jeton dans le store, aussi longtemps que la page n'est pas fermée ou rechargée, nous avons le jeton.
```

Comment:
Est ce que une traduction pour commit pourrait être 'persister' ? Car le mot 'acter' ne me parle pas beaucoup ?
Je proposerais ceci : 
> Nous persistons immédiatement le jeton dans le store, ainsi, nous conservons le jeton tant que la page n'est pas fermée ou rechargée.

Reply:
C'est intéressant. Dans la doc de Vuex c'est « acter » dans le « store ». Dans le sens tu signes un document officielle qui certifie que la valeur est a présent celle-ci. Comme les textes de lois etc. C'est tiré du commit de type GitHub (tu actes un intantané / to commit a commit).

Faut en discuter là bas (dans Vuex) du coup !